src/unpage/utils.py

```diff
@@ -497,15 +497,23 @@ async def edit_file(file_path: str | Path, editor: str | None = None) -> None:
     if not editor:
         raise ValueError("No editor specified")
 
-    editor_path = shutil.which(editor)
+    # Split editor command into command and arguments
+    editor_parts = editor.strip().split()
+    editor_cmd = editor_parts[0]
+    editor_args = editor_parts[1:]
```

Comment:
Two little things:

1. I think we should use [`shlex.split`](https://docs.python.org/3/library/shlex.html#shlex.split) instead of the string split, since it's purpose-built for this
2. In Python 3.5+, you can use arg unpacking  with the `*` operator to make splits like this a little cleaner


```suggestion
    editor_cmd, *editor_args = shlex.split(editor)
```